packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -306,6 +303,7 @@ class SortedTable extends Component {
 
     const state = (this.state = {
       all: false, // whether all items are selected (accross pages)
+      itemsPerPage: +defined(cookies.get(props.location.pathname + props.stateUrlParam), DEFAULT_ITEMS_PER_PAGE),
```

Comment:
Is it an issue that, for instance, 2 SRs will have 2 different settings due to the ID in the pathname?